integration-tests/sessions_test.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from selenium import webdriver
+
+LOCAL_HOST = 'http://localhost:3000/'
+
+
+class SignupTest(unittest.TestCase):
+    def test_signup(self):
+        driver = webdriver.Chrome()
+        driver.get(LOCAL_HOST)
+
+        self.assertEqual(driver.title, "Kvasir Movies")
+        driver.find_element_by_class_name("signup").click()
+
+        email_input = driver.find_element_by_name('email')
+        email_input.clear()
+        email_input.send_keys('naomi@jacobs.com')
+        password_input = driver.find_element_by_name('password')
+        password_input.clear()
+        password_input.send_keys('super secret')
+        driver.close()
```

Comment:
I might just not be familiar w/ the framework, but what's the rest of the test for w/o assertions? See if anything raises a python-side error?

Reply:
Yeah, before we have a test db set up, I don't want to actually run the test cause it'll add data to my db. Ideally, we would go through the actual entire signup flow, but until then, this tests that
- the home page renders
- clicking signup brings you to the signup page
- both inputs exist and are type-able

It's not super useful right now, but once I get the rest of the test scaffolding set up, it'll be more fleshed out. This was mostly meant to be a functional example to make sure the driver worked and to get it integrated with travisCI

Reply:
Sounds good. So IIUC, we never need to assert anything, b/c this is checking that nothing throws an error while on this path?